packages/account/src/Components/additional-kyc-info-modal/__test__/form-config.spec.tsx

```diff
@@ -0,0 +1,66 @@
+import { getFormFieldsConfig, getFormConfig, TFields } from '../form-config';
+
+const mockAccountSettings = {
```

Comment:
These are the shared ones between two, so we need these here, however I'll add the types 👍

Reply:
then maybe it would be better to have one describe wrapper? I believe it is ok practice to organise test cases in such way, when we need  more than one describe for scoping different cases